submitter/cli.py

```diff
@@ -1,10 +1,96 @@
+import boto3
 import click
 
+from submitter.sample_data import sample_data
 
-@click.command()
-def hello():
-    click.echo("Hello World!")
 
+@click.group()
+def main():
+    pass
 
-if __name__ == "__main__":
-    hello()
+
+@main.command()
+def start():
+    click.echo("Processing starting")
+    message_loop()
+
+
+def message_loop():
```

Comment:
I would move these three functions to a separate module named sqs or similar since they're all focused on the SQS interactions. We generally like to keep the cli module limited to just the cli commands.